package.json

```diff
@@ -41,6 +41,6 @@
     ]
   },
   "devDependencies": {
-    "cypress": "^9.5.4"
+    "cypress": "^9.6.0"
```

Comment:
Was there a reason you changed the cypress version?

Reply:
I installed Cypress thinking it hadn't been which must have changed the versions. I've changed them around.